OHHTTPStubs/Sources/Swift/OHHTTPStubsSwift.swift

```diff
@@ -161,8 +191,16 @@ public func isPath(path: String) -> OHHTTPStubsTestBlock {
  * - Note: URL paths are usually absolute and thus starts with a '/' (which you
  *         should include in the `path` parameter unless you're testing relative URLs)
  */
-public func pathStartsWith(path: String) -> OHHTTPStubsTestBlock {
-    return { req in req.URL?.path?.hasPrefix(path) ?? false }
+public func pathStartsWith(_ path: String) -> OHHTTPStubsTestBlock {
+#if swift(>=3.0)
+    return { req in req.url?.path.hasPrefix(path) ?? false }
+#else
+#if os(tvOS)
+    return { req in req.url?.path?.hasPrefix(path) ?? false }
+#else
+    return { req in req.url?.path?.hasPrefix(path) ?? false }
+#endif
```

Comment:
Might be a bad merge. But this was there because the legacy-swift setting for the tvOS target was YES. I fixed this in my fork, so this `#if` shouldn't be there and the legacy-swift setting should be checked.